messaging/src/main/java/org/axonframework/commandhandling/gateway/IntervalRetryScheduler.java

```diff
@@ -63,6 +63,7 @@ protected long computeRetryInterval(CommandMessage commandMessage,
      * Instantiate a Builder to be able to create a {@link IntervalRetryScheduler}.
      * <p>
      * The default for {@code retryInterval} is set to 100ms, while {@code maxRetryCount} gets a single retry.
+     * The default for {@code nonTransientFailurePredicate} is set to {@link DefaultNonTransientPredicate}.
```

Comment:
Nit: Other mentions of default settings are paired into a single sentence particularly.
So, something like this:

```suggestion
     * The default for {@code maxRetryCount} is set to a single retry, the {@code backoffFactor} defaults to 100ms and the {@code nonTransientFailurePredicate} defaults to {@link DefaultNonTransientPredicate}.
```

Reply:
By the way, if we rename the predicate, this bit of documentation should also change.

Reply:
And lastly, not just the `builder()` method, but also the `class Builder` should contain this bit of documentation.